src/app/authentication/forgot-password.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm }    from '@angular/forms';
+import { AuthService } from '../services/auth.service';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+
+@Component({
+    selector: 'forgot-password',
+    templateUrl: './forgot-password.component.html',
+    styleUrls: ['./login.component.scss']
+})
+export class ForgotPasswordComponent implements OnInit {
+
+    resetForm : FormGroup;
+    errorMessage : String;
+    successMessage : String;
+
+    constructor(private _authService: AuthService, fb: FormBuilder) {
+        this.resetForm = fb.group({
+            'email' : [null, Validators.compose([Validators.required, Validators.pattern("[^ @]*@[^ @]*")])]
```

Comment:
it's good that you are checking email format in the frontend, but it's better (for security) to validate in the backend. remember that frontend is transparent, everyone can see the code & logic for frontend